CSS/style.css

```diff
@@ -35,6 +41,75 @@
     font-size: 14px;
     line-height: 20px;
     text-decoration: none;
+
+}
+.nav-bar-badge {
+    display: flex;
+    gap: 12px;
+    list-style: none;
+
+}
+
+.hero-section {
```

Comment:
In .hero-section you can add `min-height: 100vh` to set the hero page to 100% of the viewport height.  Using it with only `height` is not good because it's fixed size it will always be 100vh for example when your elements exceeds 100vh it might look broken but when you make `min-height` it's minimum height will be 100vh and if exceeds it will not be broken.